lib/voyager/clients/microsoft_graph_client.rb

```diff
@@ -51,22 +51,38 @@ def my_drives
       get('/me/drives')
     end
 
-    def my_drive
-      get('/me/drive')
+    def my_drive_children
+      get('/me/drive/root/children')
     end
 
     def my_drive_items(item_id = 'root', options = '')
       get("/me/drive/#{item_id}/children" + options)
     end
 
-    def drive(drive_id, query_opts = '')
+    def following
+      get('/me/drive/following')
+    end
+
+    def shared
+      get('/me/drive/sharedWithMe')
```

Comment:
same here.  I wouldn't be offended if we called this (`def drives_shared_with_me` or `def drives_shared` is fine too)